common/types/hashes.go

```diff
@@ -1,110 +1,27 @@
 package types
 
 import (
-	"encoding/hex"
-	"fmt"
-	"math/big"
-	"reflect"
-
-	"github.com/spacemeshos/go-scale"
-
-	"github.com/spacemeshos/go-spacemesh/common/util"
+	"github.com/spacemeshos/go-spacemesh/common/types/primitive"
 	"github.com/spacemeshos/go-spacemesh/hash"
-	"github.com/spacemeshos/go-spacemesh/log"
 )
 
-const (
-	// Hash32Length is 32, the expected length of the hash.
-	Hash32Length = 32
-	hash20Length = 20
-	hash12Length = 12
+// Reexports.
+type (
+	Hash20 = primitive.Hash20
+	Hash32 = primitive.Hash32
 )
```

Comment:
Why move the type definitions to a different package and alias them here? Can't they stay here? I assume this is because of the `wire` package? This problem goes away if the types in the `wire` package are moved to the `activation` package (or a sub package of it)

Reply:
reverted